themes/classic/modules/contactform/views/templates/widget/contactform.tpl

```diff
@@ -98,9 +99,10 @@
         {/if}
 
         <div class="form-group row">
-          <label class="col-md-3 form-control-label">{l s='Message' d='Shop.Forms.Labels'}</label>
+          <label class="col-md-3 form-control-label" for="message">{l s='Message' d='Shop.Forms.Labels'}</label>
           <div class="col-md-9">
             <textarea
+              id="message"
```

Comment:
I'm just afraid this id might be a bit too generic (it can easily be used with this same value in another place in the page, or another module) It would be less risky to prefix all the IDs with some context like `id='contact-message'` `id='contact-id-order'` But I guess it's fine for now

Reply:
I agree and fixed this one, but others can't be changed overwise it's a BC